lib/ansible/cli/__init__.py

```diff
@@ -353,6 +353,8 @@ def base_parser(usage="", output_opts=False, runas_opts=False, meta_opts=False,
                 help="specify extra arguments to pass to scp only (e.g. -l)")
             connect_group.add_option('--ssh-extra-args', default='', dest='ssh_extra_args',
                 help="specify extra arguments to pass to ssh only (e.g. -R)")
+            connect_group.add_option('--nspawn-extra-args', default='', dest='nspawn_extra_args',
+                help="specify extra arguments to pass to systemd-nspawn only (e.g. --bind)")
```

Comment:
i prefer to not have extra command line args (really want to remove ssh-extra and use a --connection-extra instead).